schema/src/lib.rs

```diff
@@ -262,6 +265,52 @@ macro_rules! impl_schema_int {
 
 impl_schema_int!(u128, u64, u32, u16, u8, i128, i64, i32, i16, i8);
 
+impl TypeId for NonZeroU32 {
```

Comment:
So these impls for `NonZeroU32` are useless for now, right? Because there are no `NonZeroU32` in schema

Reply:
Not useless. The implementation is such that NonZeroU32/NonZeroU64 are mapped into u32/u64 respectively. That is why there is no change in the schema.json

Reply:
I decided to add implementation for `NonZeroU32` in one go with `NonZeroU64` but it's not used in the schema at the moment.